themes/yas/login/resources/img/yaslogo.pngZone.Identifier

```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]
```

Comment:
the name of this file is very strange. Can you rename it ?

Reply:
I dont what is this file too. It's auto generate when i add logo img into folder, i will check again.